rio_tiler/constants.py

```diff
@@ -2,10 +2,15 @@
 
 import multiprocessing
 import os
+from typing import Tuple, Union
 
 import morecantile
 from rasterio.crs import CRS
 
+NumType = Union[float, int]
+BBox = Tuple[NumType, NumType, NumType, NumType]
```

Comment:
Is it important to also let these be `int`?

Reply:
I don't know 🤷‍♂️